s3file/storages.py

```diff
@@ -52,6 +54,27 @@ class bucket:
         name = "test-bucket"
 
 
+class S3OptimizedMockStorage(S3OptimizedUploadStorage):
+    created_objects = {}
+
+    def _compress_content(self, content):
+        return content
+
+    class bucket:
+        name = "test-bucket"
+
+        class Object:
+            def __init__(self, key):
+                self.key = key
+                self.copy_from_bucket = None
+                self.copy_from_key = None
+                S3OptimizedMockStorage.created_objects[self.key] = self
+
+            def copy(self, s3_object, ExtraArgs):
+                self.copy_from_bucket = s3_object["Bucket"]
+                self.copy_from_key = s3_object["Key"]
+
+
```

Comment:
Wait a sec, I why do we need a second mock storage?
This `S3MockStorage` is not for testing purposes within this package, but a public API for local development. It's needed to test things like the JavaScript events.

If you only need a mock for testing, please move it to the `./tests` folter.

Reply:
Ah, I see. Makes sense. Will re-check if it's even needed but I thought so as there is no way we could setup something like a proper S3 API mock or so?

Reply:
Moved this mock storage to the tests.